src/GraphQL/Execution/ExecutionOptions.cs

```diff
@@ -12,22 +12,39 @@
 
 namespace GraphQL
 {
+    /// <summary>Configuration options to be passed to <see cref="IDocumentExecuter"/> to execute a query</summary>
     public class ExecutionOptions : IProvideUserContext
     {
+        /// <summary>Schema of graph to use; required<br/><br/>Schema will be initialized if has not yet been initialized.</summary>
         public ISchema Schema { get; set; }
+
+        /// <summary>Object to pass to the <see cref="IResolveFieldContext.Source"/> property of first-level resolvers</summary>
         public object Root { get; set; }
+
+        /// <summary>GraphQL query to parse and execute; required</summary>
         public string Query { get; set; }
+
+        /// <summary>GraphQL query operation name; defaults to first operation defined in query</summary>
         public string OperationName { get; set; }
+
+        /// <summary>Parsed GraphQL request; will parse <see cref="Query"/> if not set</summary>
         public Document Document { get; set; }
+
+        /// <summary>Input variables to GraphQL request</summary>
         public Inputs Inputs { get; set; }
-        
+
+        /// <summary><see cref="System.Threading.CancellationToken">CancellationToken</see> of request; optional</summary>
         public CancellationToken CancellationToken { get; set; }
 
-        /// <summary>
-        /// Note if not set then standard list of validation rules will be used.
-        /// </summary>
+        /// <summary>Validation rules to be used by the <see cref="IDocumentValidator"/>; defaults to standard list of of validation rules - see <see cref="DocumentValidator.CoreRules"/></summary>
         public IEnumerable<IValidationRule> ValidationRules { get; set; }
 
+        /// <summary>
+        /// User-defined variables to be passed to the field resolvers.<br/>
```

Comment:
```suggestion
        /// User-defined context to be passed to the field resolvers.<br/>
```
The word "variables" can be confusing to reader as there is `Inputs` property for variables.